### PR TITLE
refactor: Improve typings for webhook methods (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

File: packages/workflow/src/Interfaces.ts [hunk 1]
```diff
@@ -1172,9 +1174,6 @@ export interface INodeType {
 	poll?(this: IPollFunctions): Promise<INodeExecutionData[][] | null>;
 	trigger?(this: ITriggerFunctions): Promise<ITriggerResponse | undefined>;
 	webhook?(this: IWebhookFunctions): Promise<IWebhookResponseData>;
-	hooks?: {
-		[key: string]: (this: IHookFunctions) => Promise<boolean>;
-	};
```
Comment (maspio) [inline packages/workflow/src/Interfaces.ts:1177]: Interesting... do you know why we had hooks here? Can't find anything about it.

Comment (netroy) [reply packages/workflow/src/Interfaces.ts:1177]: this was never used anywhere, from what I can tell. possibly a relic from a long time ago.

**Review by maspio — APPROVED** [review]
Looks good to me

**Comment by janober** [comment]
Got released with [`n8n@0.221.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.221.2)

**Comment by janober** [comment]
Got released with [`n8n@0.221.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.221.2)

**Pull request merged**